notifications.bs

```diff
@@ -278,7 +278,9 @@ language tag. Validity or well-formedness are not enforced. [[!LANG]]
   </ol>
 
   <li><p>If the notification platform supports sounds, <a>fetch</a>
-  <var>notification</var>'s <a>sound URL</a>, if <a>sound URL</a> is set.
+  <var>notification</var>'s <a>sound URL</a>, if <a>sound URL</a> is set,
+  the <var>notification</var>'s <a>renotify preference flag</a> has been set
+  or the <var>notification</var> is not being replaced.
```

Comment:
1. The conditional is somewhat unclear.
2. "being replaced" would need to be defined. Presumably using the same primitive we use to actually replace a notification. (Yay, yet more abstract algorithms.)